llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -9562,8 +9562,12 @@ void VPWidenMemoryInstructionRecipe::execute(VPTransformState &State) {
   InnerLoopVectorizer::VectorParts BlockInMaskParts(State.UF);
   bool isMaskRequired = getMask();
   if (isMaskRequired)
```

Comment:
nit: worth retaining the comment below, perhaps here, that reversal is restricted to required masks because `// Reverse of a null all-one mask is a null mask.`

Reply:
Done in 2a9aed17302c8363883a228d35f398110b575db3, thanks!